python_modules/libraries/dagster-powerbi/dagster_powerbi_tests/test_asset_specs.py

```diff
@@ -67,17 +71,31 @@ def test_translator_dashboard_spec(workspace_data_api_mocks: None, workspace_id:
     }
 
 
+@lazy_definitions
+def state_derived_defs_two_workspaces() -> Definitions:
+    resource = PowerBIWorkspace(
+        credentials=PowerBIToken(api_token=EnvVar("FAKE_API_TOKEN")),
+        workspace_id="a2122b8f-d7e1-42e8-be2b-a5e636ca3221",
+    )
+    resource_second_workspace = PowerBIWorkspace(
+        credentials=PowerBIToken(api_token=EnvVar("FAKE_API_TOKEN")),
+        workspace_id="c5322b8a-d7e1-42e8-be2b-a5e636ca3221",
+    )
+    return Definitions.merge(
+        resource.build_defs(),
+        resource_second_workspace.build_defs(),
+    )
+
+
```

Comment:
definitions for these tests are now inlined instead of living in separate files

Reply:
I guess it no longer does-- at one point `test_two_workspaces` used to load the defs at one point with a `ReconstructibleRepository` like the `test_state_derived_defs()` test below does, and `@lazy_definitions` is required in that case to identify the value as a loadable target.

But this test no longer does that. Still I think it's useful just for visual clarity to mark definitions-returning functions in this way, even if they aren't always loaded through the "loadable target" pathway that requires it.

Reply:
> Still I think it's useful just for visual clarity to mark definitions-returning functions in this way, even if they aren't always loaded through the "loadable target" pathway that requires it.

Disagree. When I see this I assume that some test or some code relies on the loadable target behavior. We have bare functions that return `Definitions` all over the place in a world with `Definitions.merge`.